MovieMVP/ListMovieModule/View/ListMoviesViewController.swift

```diff
@@ -109,34 +110,34 @@ final class ListMoviesViewController: UIViewController {
         setupActiveButton(pressedButton: sender)
         switch sender.tag {
```

Comment:
```suggestion
        sender.tag {
```
вся логика в presenter. Tag передавай в presenter и там switch